### DISCUSSION TITLE
Cursor Pagination

### DISCUSSION THREAD

**Discussion by abdulrahimiliasu** [prisma/prisma#15518] [Q&A] [answered]
I just want to confirm if this two queries are the same. i.e using `where` and `cursor`

```
.findMany({
            take: 100,
            skip: 1, // Skip the cursor
            where: {
                id: cursorId
            },

            orderBy: {
                date: 'desc'
            }
        }
```

and 

```
.findMany({
            take: 100,
            skip: 1, // Skip the cursor
            cursor: {
                id: cursorId
            },
            orderBy: {
                date: 'desc'
            }
        }
```

**Comment by austincrim** [comment, marked as answer]
Hey @abdulrahimiliasu 👋,

Those two queries **are not** equivalent as the first query will return the first 100 records which have an `id` field **that is equal to** `cursorId` and the second query will return the first 100 records **after the given `cursorId`**.

You can [read more about pagination in our docs](https://www.prisma.io/docs/concepts/components/prisma-client/pagination#cursor-based-pagination).

**Reply by abdulrahimiliasu** [reply]
I see thanks @austincrim

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
